Front-End/challenge2/script.js

```diff
@@ -5,3 +5,104 @@ TO-DO:
 - The calculator should be completely functional
 
 */
+
+"use strict"
+
+const wrapper = document.querySelector('.wrapper')
+
+let numberStr = ''
+let viewString = ''
+let operations = ['+', '-', 'X', '/']
```

Comment:
for all variables that are not intended to reassign its value please use `const`

Reply:
Fixed